src/browser/fixture.js

```diff
@@ -29,22 +29,29 @@ function getComposedChildren(node) {
 
 }
 
-async function waitForElem(elem) {
+async function waitForElem(elem, opts) {
 
 	const update = elem?.updateComplete;
 	if (typeof update === 'object' && Promise.resolve(update) === update) {
 		await update;
 		await nextFrame();
 	}
 
+	if (opts.awaitLoadingComplete && typeof elem?.getLoadingComplete === 'function') {
```

Comment:
What are the potential use cases for not wanting to wait for `getLoadingComplete`, if it exists?  To vdiff skeletons?

Reply:
Yeah that was the only use case I could think of, which is why I defaulted this to ON.